charmcraft.yaml

```diff
@@ -43,6 +43,17 @@ parts:
     override-pull: |
       curl -L -O https://github.com/canonical/cos-tool/releases/latest/download/cos-tool-${CRAFT_TARGET_ARCH}
       chmod +x cos-tool-*
+  lokitool:
+    plugin: dump
+    source: .
+    build-packages:
+      - curl
+      - unzip
+    override-pull: |
+      curl -fLo lokitool.zip https://github.com/grafana/loki/releases/download/v3.2.1/lokitool-linux-${CRAFT_TARGET_ARCH}.zip
+      unzip lokitool.zip && rm lokitool.zip
+      mv lokitool-linux-${CRAFT_TARGET_ARCH} lokitool
+      chmod +x lokitool
```

Comment:
Just a thought:
- Should charm code be pulling lokitool from the rock? This way compatibility is likely less brittle.
- Should we keep lokitool in the workload container, and instead pebble push & exec?

Reply:
The rock is the Nginx rock, not the Loki one, because this is the coordinator, so compatibility-wise things are the same.